changelogs/unreleased/Update_python_version_for_iso8 copy.yml

```diff
@@ -0,0 +1,5 @@
+description: Update python lower bound to 3.12 for ISO8
+change-type: major
+destination-branches: [master]  
+sections:
+  feature: "{{description}}"
```

Comment:
We must also mention that we use the native docker containers.

Reply:
Then I will need two changelog files, I presume?

Reply:
Can I defer this to https://github.com/inmanta/inmanta-core/issues/8304

Reply:
You can have a `feature` and an `upgrade-note` section in one changelog file. But I am fine with deferring this to the referenced ticket.